woocommerce/woocommerce
# [Product Block Editor] Show onboarding tour within description block editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Show onboarding tour within description block editor

Closes #39131 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at wp-admin/admin.php?page=wc-settings&tab=advanced&section=features 
4. Go to Products > Add new
5. Open the developer console and paste `wp.data.dispatch('core/preferences').set('core/edit-post', 'welcomeGuide', true);`
6. Click on "Add Description"
7. You should see the same welcome guide. (You can compare by opening the post editor, clicking the 3 dots menu, and clicking 'Welcome guide')
8. Test the functionality, by pressing next, previous, and the "Get started" button
9. The Get Started button and the "X" buttons should close the guide.
10. Refresh the page and open the description modal again. It should not re-appear.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
